fuzz/fuzz_targets/block.rs

```diff
@@ -71,7 +70,7 @@ fuzz_target!(|bytes| {
 
     queue_evt.write(77).unwrap(); // Rings the doorbell, any byte will do.
 
-    wait_queue_event_processed(queue_evt);
+    let _ = block.wait_for_all_events_once();
```

Comment:
If this is accepted, please squash this commit with the one introducing `wait_queue_event_processed()`.